iree/compiler/Dialect/Modules/VMVX/Transforms/Passes.cpp

```diff
@@ -75,8 +75,8 @@ static void buildVectorVMVXTransformPassPipeline(OpPassManager &passManager) {
 
   // Tiling and distribution.
   nestedModulePM.addNestedPass<FuncOp>(createCanonicalizerPass());
-  nestedModulePM.addNestedPass<FuncOp>(
-      createLinalgTileAndVectorizeWorkgroupsPass());
+  // nestedModulePM.addNestedPass<FuncOp>(
```

Comment:
Why this needs to be commented ?

Reply:
Getting a failure on VMVX backend. Something is not setup correctly. After the CPU backend moves to use dynamic pass pipelines, the way this is done in VMVX can be cleaned up as well. So disabling this till that PR lands. See https://github.com/google/iree/issues/5925 . Ill link that issue here.